airbyte-integrations/connectors/source-hubspot/source_hubspot/source.py

```diff
@@ -97,7 +100,15 @@ def get_granted_scopes(self, authenticator):
         try:
             access_token = authenticator.get_access_token()
             url = f"https://api.hubapi.com/oauth/v1/access-tokens/{access_token}"
-            response = requests.get(url=url)
+            error_resolution = ErrorResolution(ResponseAction.RETRY, FailureType.system_error, "Internal error attempting to get scopes.")
```

Comment:
Should we consider this as transient errors instead of system ones?

Reply:
done